modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -24,7 +24,6 @@
 import java.util.Date;
 import java.util.concurrent.TimeUnit;
 
-import static com.google.common.base.Preconditions.checkState;
```

Comment:
since it's the only place where we use Guava, I removed it from `selenium` module and inlined the implementation of `checkState`, so that we don't have to shade guava into the module